content/posts/fleek-network-running-a-node-in-a-docker-container.md

```diff
@@ -107,9 +114,21 @@ For more examples and ideas, visit:
 
 Run all the commands above in your terminal, to confirm 👍 everything's working before proceeding to the next steps, please!
 
-## Build the Docker image
+## Image
+
+A Docker image is a read-only template with instructions for creating a Docker container, like a template. Docker images also act as a the starting point when using Docker. 
+
+The starting point in for our use-case is a Dockerfile, where all those "template instructions" are declared.
```

Comment:
I think it should be either "The starting point in our use-case is a Dockerfile..." or "The starting point for our use-case is a Dockerfile...", so either removing "in" or "for"